pages/common/qmmp.md

```diff
@@ -0,0 +1,32 @@
+# qmmp
+
+> A small and light audio player.
+> More information: <http://qmmp.ylsoftware.com/>.
+
+- Open qmmp:
```

Comment:
Is a GUI?
```suggestion
- Launch qmmp:
```

Reply:
is an interactive mode?
```suggestion
- Start interactive mode:
```

Reply:
> Is a GUI?

I installed this just now. Can confirm is GUI and this command opens GUI.